storefront/common/components/AuthenticationInfo.tsx

```diff
@@ -20,10 +20,15 @@ export default function AuthenticationInfo() {
     return await res.json();
   }
 
+  async function moveCart() {
+    await fetch(`/move-cart`);
+  }
+
   useEffect(() => {
     getAuthenticationInfo().then((data) => {
       setAuthenticatedInfoVm(data);
     });
+    moveCart();
```

Comment:
You think we should check and call moveCart only when use is authenticated?

Reply:
I have updated source code to check authentication before call move-cart api, please review again